config/locales/instructions/en.yml

```diff
@@ -2,7 +2,21 @@
 en:
   instructions:
     2fa:
-      confirm_code: We sent it to %{number}.
+      sms:
+        confirm_code: We sent it in a text message to %{number}. Need another code?
+          %{resend_code_link}. Message rates may apply.
+        fallback: If you can't get text messages right now, you can get a passcode
+          via %{link}
+      voice:
+        confirm_code: We just called you at %{number}. Want us to call you again?
+          %{resend_code_link}
+        fallback: If you can't take a phone call right now, you can get a passcode
+          via %{link}
+      totp:
+        confirm_code: Enter the code from your authenticator app. If you have several accounts
```

Comment:
- since this contains HTML it should definitely be named with `_html`
- optionally, we may want to move the `<strong>` tags to `content_tag` interpolations in the helpers? -- as previously mentioned, I've had issues with HTML tags coming back broken through translation pipelines for other languages